js/data/locale/af/NA/ctrynames.json

```diff
@@ -211,14 +210,13 @@
     "Pseudoaksente": "XA",
     "Pseudotweerigting": "XB",
     "Puerto Rico": "PR",
-    "Réunion": "RE",
     "Roemenië": "RO",
     "Rusland": "RU",
     "Rwanda": "RW",
+    "Réunion": "RE",
     "Salomonseilande": "SB",
     "Samoa": "WS",
     "San Marino": "SM",
-    "São Tomé en Príncipe": "ST",
```

Comment:
The difference for most of these files is that it is sorting differently.... This should not happen.

Reply:
The difference is that json-stable-stringify uses a different comparison function. It was already sorted before, so we just need to pass in the comparison function to get it to sort the same way it did before.

Reply:
When you see the `regionnames.json` file that is generated with the tool code as well, I can see the data is not sorted.
i.e.
current data.
```
"AD" : [
  "name":"Canillo",
  "code": "02"
]
```
I expect to get a result like below.
```
"AD" : [
  "code": "02"
  "name":"Canillo",
]
```